gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -102,7 +108,10 @@ class {{ service.client_name }}Meta(type):
 
 
 class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
-    """{{ service.meta.doc|rst(width=72, indent=4) }}"""
+    """{{ service.meta.doc|rst(width=72, indent=4) }}{% if service.version|length %}
+    This class implements API version {{ service.version }}."""
+    {% else %}"""
```

Comment:
See my comment in the ads template about removing the `else`

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/1e6e96e0654f5c941cf569a0d97f866946cc68e3